src/utils/__tests__/error-utils.unit.spec.js

```diff
@@ -0,0 +1,118 @@
+import { failAction } from '../error-utils.js'
+
+describe('error-utils.unit', () => {
+  describe('failAction', () => {
+    let mockH
+    const defaultMockError = {
+      details: [
+        {
+          message: '"name" is required',
+          path: ['name']
+        },
+        {
+          message: '"age" must be a number',
+          path: ['age']
+        }
+      ],
+      _original: {
+        name: '',
+        age: 'not-a-number'
+      }
+    }
```

Comment:
Please amend defaultMockError so it's a function that returns the object, rather that being an object we reuse.